Example/PrimerSDK/Info.plist

```diff
@@ -56,5 +56,9 @@
 	<array>
 		<string>UIInterfaceOrientationPortrait</string>
 	</array>
+    <key>LSApplicationQueriesSchemes</key>
+    <array>
+        <string>bankid</string>
```

Comment:
does this mean every merchant needs to add this to their app?

Reply:
yeah